### ISSUE TITLE
Evergreen Campaign Support

### ISSUE THREAD

**Issue by JorgeMucientes** [woocommerce/woocommerce-android#12176]
Main issue to keep track of Blaze i4 tasks on Android

```[tasklist]
### UI
- [ ] https://github.com/woocommerce/woocommerce-android/issues/12248
- [ ] https://github.com/woocommerce/woocommerce-android/issues/12445
```

```[tasklist]
### Logic
- [ ] https://github.com/woocommerce/woocommerce-android/issues/12390
- [ ] https://github.com/woocommerce/woocommerce-android/issues/12446
- [ ] https://github.com/woocommerce/woocommerce-android/issues/12453
```

**Comment by dangermattic** [comment]
Thanks for reporting! 👍

**Comment by JorgeMucientes** [comment]
Support to create Evergreen campaigns will be enabled starting from app `20.2`. 
However the final UI for the Blaze budget screen will be released in the next version `20.3`
